src/util.cpp

```diff
@@ -889,6 +889,99 @@ QSize CMinimumStackedLayout::sizeHint() const
 * Other Classes                                                                *
 \******************************************************************************/
 // Network utility functions ---------------------------------------------------
+bool NetworkUtil::ParseNetworkAddressString ( QString strAddress, QHostAddress& InetAddr, bool bEnableIPv6 )
+{
+    // try to get host by name, assuming
+    // that the string contains a valid host name string or IP address
+    const QHostInfo HostInfo = QHostInfo::fromName ( strAddress );
+
+    if ( HostInfo.error() != QHostInfo::NoError )
+    {
+        // qInfo() << qUtf8Printable ( QString ( "Invalid hostname" ) );
+        return false; // invalid address
+    }
+
+    foreach ( const QHostAddress HostAddr, HostInfo.addresses() )
+    {
+        // qInfo() << qUtf8Printable ( QString ( "Resolved network address to %1 for proto %2" ) .arg ( HostAddr.toString() ) .arg (
```

Comment:
Although unrelated, how do we handle debug output in other places at the moment? Commented code, QDebug()?

Reply:
Note: This code is unchanged -- it only got moved around and therefore appears in this PR.

I don't see a clear logging policy in the code base, at least not as clear as other (server) software. We shouldn't have commented out code. Instead, we should make proper use of different debug levels and allow controlling them via command line. But yeah, out of scope here...

Reply:
We can leave it like this for now and look at it in a later refactoring.